File: 프로그래머스/0/181904. 세로 읽기/세로 읽기.kt
class Solution {
    fun solution(my_string: String, m: Int, c: Int): String {
        val answer = mutableListOf<Char>()
        val length = my_string.length
        
        for (i in c-1 until length step m) {
            answer.add(my_string[i])
        }
        return answer.joinToString("")
    }
}

File: 프로그래머스/0/181904. 세로 읽기/세로 읽기.js
function solution(my_string, m, c) {
    let answer = [];
    const length = my_string.length;
    
    for (let i = c - 1; i < length; i += m) {
        answer.push(my_string[i]);
    }
    
    return answer.join('');
}